spark/src/main/java/io/delta/dynamodbcommitstore/DynamoDBCommitOwnerClient.java

```diff
@@ -655,16 +709,27 @@ private void tryEnsureTableExists() throws IOException {
                 }
             }
             if (status.equals("ACTIVE")) {
+                if (created) {
+                    LOG.info("Successfully created DynamoDB table `{}`", managedCommitsTableName);
```

Comment:
Could we add couple of log statements in Commit method as well.

- Attempting to commit <> via DynamoDB
- Commit <> done successfully. Backfilling commit files.